Sources/UI/Experiences/Models/Experience.swift

```diff
@@ -61,7 +61,7 @@ extension Experience: Decodable {
 // MARK: AttributeRepresentable
 
 extension Experience: AttributeRepresentable {
```

Comment:
`AttributeRepresentable` has been removed in this PR. You'll need to update this to match the approach you took in the `Beacon` and `Geofence` classes.